app/uk/gov/hmrc/hec/testonly/services/TaxCheckService.scala

```diff
@@ -61,7 +61,8 @@ class TaxCheckServiceImpl @Inject() (
         saveTaxCheckRequest.expiresAfter,
         saveTaxCheckRequest.createDate,
         false,
-        None
+        None,
+        saveTaxCheckRequest.fileCorrelationId
```

Comment:
I think it'd make sense for this to be `None` and not be a parameter in `SaveTaxCheckRequest` - it's unlikely that we'd be able to use it in a request I think and be useful given how the file correlation id generation happens in the scheduled job